nano/rpc_test/rpc.cpp

```diff
@@ -142,7 +142,15 @@ boost::property_tree::ptree wait_response (nano::system & system, std::shared_pt
 void check_block_response_count (nano::system & system, std::shared_ptr<nano::rpc> const & rpc, boost::property_tree::ptree & request, uint64_t size_count)
 {
 	auto response (wait_response (system, rpc, request));
-	ASSERT_EQ (size_count, response.get_child ("blocks").front ().second.size ());
```

Comment:
Looks like this used to be UB before with an empty blocks and then calling `.second` on the `.front()`. Did it ever crash though? 🤔

Reply:
Yes, it was periodically crashing or giving uninitialized results.